conductr_cli/resolvers/resolvers_util.py

```diff
@@ -9,7 +8,6 @@ def is_local_file(uri, require_bundle_conf):
     return parsed.scheme == 'file' and os.path.exists(parsed.path) and (
         os.path.isfile(parsed.path) or (
             os.path.isdir(parsed.path) and
-            detect_format_dir(parsed.path) is not None and
```

Comment:
Thanks for catching this. I'll run a manual test for this use case to see how it behaves.